setup.py

```diff
@@ -360,15 +360,9 @@ def run(self):
               "fixtures",
               "beautifulsoup4",
               "html5lib",
-          ],
-          "tor": [
-              "foolscap[tor] >= 0.12.5",
-              "txtorcon >= 0.17.0", # in case pip's resolver doesn't work
-          ],
-          "i2p": [
-              "foolscap[i2p] >= 0.12.6",
-              "txi2p >= 0.3.2", # in case pip's resolver doesn't work
-          ],
+          ] + tor_requires + i2p_requires,
+          "tor": tor_requires,
+          "i2p": i2p_requires,
       },
```

Comment:
Hm why is Github showing me this hunk...

Reply:
Could be that it was calculated before I merged upstream?

Reply:
It says 'outdated' now.